src/data/content/design-system/fondamenti/icone.yaml

```diff
@@ -0,0 +1,286 @@
+      navposition:
+        items:
+          - title: Ti trovi in
+            tooltip: "La sezione di Designers Italia cui appartiene questa pagina"
+            icon:
+              icon: "sprites.svg#it-folder"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Fondamenti #C
+            url: "../../" #M
+      otherInfo:
+        items:
+          - title: Rispettando
+            tooltip: "Norme e riferimenti che i contenuti di questa pagina ti aiutano a rispettare"
+            icon:
+              icon: "sprites.svg#it-pa"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Linee guida di design #C
+            url: "/norme-e-riferimenti/linee-guida-di-design" #M
+      personalInfo:
+         items: #C
+          - title: Sei
+            tooltip: "Se non ti riconosci in questi profili non preoccuparti: potrai scoprire informazioni utili per saperne di più. In alternativa, naviga la sezione superiore per approfondire il contesto"
+            icon:
+              icon: "sprites.svg#it-user"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Designer, Developer, Qui per curiosità #C
+          - title: Vuoi
+            tooltip: "Obiettivi di progettazione per cui sono utili i contenuti di questa pagina"
+            icon:
+              icon: "sprites.svg#it-flag"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Progettare e sviluppare l'interfaccia utente #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Progettazione interfaccia, Sviluppo interfaccia]
+  
+  sectionsEditorial:
+    - full: true
+      noSpace: false
+      paddingLeft: true
+      title: null
+      text: null
+      background: null
+      components:
+
+        - name: TextImageCta
+          title: A cosa servono
+          headingLevel: 2
+          text: |
+            Le icone sono elementi visivi che contengono simboli: sono cruciali per un'interfaccia di qualità e hanno lo scopo di veicolare informazioni agli utenti in modo rapido e immediato. 
+            
+            Possono rappresentare **funzioni**, **categorie**, **opzioni** e molto altro, migliorando l'interazione in termini di usabilità e accessibilità. Ciò permette agli utenti di navigare e interagire con l'applicazione o il sito web in modo più intuitivo. 
+            
+            Le icone devono essere facilmente riconoscibili e comprensibili, garantendo un design coerente nell'ambito della soluzione digitale. 
+        
+        - name: TextImageCta
+          title: Come usarle
+          headingLevel: 2
+          text: |
+            Usa le icone solo quando è strettamente necessario e per i seguenti casi d'uso:
```

Comment:
Va recuperato il disclaimer/indicazione di usarle sempre associate a etichetta e/o tooltip ([vedi manuale](https://docs.italia.it/italia/designers-italia/manuale-operativo-design-docs/it/versione-corrente/doc/esperienza-utente/progettare-e-costruire-in-alta-fedelta.html#le-icone)) e/o in casi eccessionali da testo solo per screen reader (da aggiungere).

Reply:
@Fupete intendi, aggiungere il link al paragrafo dedicato delle icone?

Reply:
No specificherei anche qui proprio la stessa cosa già dichiarata nel manuale. Il rischio di trovarsi icone senza label o almeno testi alternativi nelle interfacce è sennò troppo alto.

Reply:
ok allora cassiamo

Reply:
ho proposto una review